font.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'pt_serifbold';
+  src: url('fonts/ptserif-bold-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'pt_serifbold_italic';
+  src: url('fonts/ptserif-bolditalic-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'pt_serifitalic';
+  src: url('fonts/ptserif-italic-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'pt_serif';
+  src: url('fonts/ptserif-regular-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Goal_tereshchenko/pullRequest?prid=6223281)